src/components/Markdown/Markdown.js

```diff
@@ -3,18 +3,20 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components'
```

Comment:
@dizzypaty We may want to think about how to represent headings inside the markdown content of each app's description.

This is what some descriptions currently look like (the internal heading is quite awkward, since it's larger than the outside heading):

<img width="450" alt="Screen Shot 2019-08-26 at 7 12 44 PM" src="https://user-images.githubusercontent.com/4166642/63708745-962f1280-c835-11e9-8126-90690cf30e08.png">

Should these internal headings be `label2` as well? (@AquiGorka we should change the description.md of the apps to use actual headings, and then apply the correct style here)

Reply:
Done here and [there](https://github.com/aragon/aragon-apps/pull/951)